core/src/main/scala/tofu/Errors.scala

```diff
@@ -116,7 +116,14 @@ trait Errors[F[_], E] extends Raise[F, E] with Handle[F, E] with ErrorsTo[F, F,
     recoverWith(fa)(pf.andThen(raise[A] _))
 }
 
-object Errors extends DataEffectComp[Errors]
+object Errors extends DataEffectComp[Errors] {
+
+  trait Companion[E] {
+    type Raise[F[_]]  = tofu.Raise[F, E]
+    type Handle[F[_]] = tofu.Handle[F, E]
```

Comment:
WDYT on names as in [the article by Pavel](https://scala.monster/train-station-tofu/#improved-error-handling):
```scala
type Handling[F[_]] = Handle[F, DepartureError]
type Raising[F[_]]  = Raise[F, DepartureError]
```

It is easier to read and more convenient for me.

Reply:
I also support different naming to avoit conflicts